sdk/spring/spring-cloud-azure-actuator-autoconfigure/CHANGELOG.md

```diff
@@ -1,14 +1,7 @@
 # Release History
 
-## 4.0.0-beta.4 (Unreleased)
+## 4.0.0 (2022-03-28)
```

Comment:
I remember some pipeline will failed if there is no content in this section.

Reply:
yes, we will link to the changelog in sdk/spring, I will update them to point to it after this pr is merged.